maps/away/ships/elyra.dm

```diff
@@ -0,0 +1,103 @@
+	icon_state = "shuttle"
+	moving_state = "shuttle_moving"
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	initial_restricted_waypoints = list(
+		"Elyran Naval Shuttle" = list("nav_hangar_elyra")
+	)
+
+	initial_generic_waypoints = list(
+		"nav_elyran_strike_craft_1",
+		"nav_elyran_strike_craft_2"
+	)
+
+/obj/effect/overmap/visitable/ship/elyran_strike_craft/New()
+	name = "ENV [pick("Persepolis", "Damascus", "Medina", "Aemaq", "New Suez", "Bursa", "Republican", "Falcon", "Gelin", "Sphinx", "Takam", "Dandan", "Anqa", "Falak", "Uthra", "Djinn", "Roc", "Shadhavar", "Karkadann")]"
+	..()
+
+/obj/effect/shuttle_landmark/elyran_strike_craft/nav1
+	name = "Elyran Naval Strike Craft - Port Side"
+	landmark_tag = "nav_elyran_strike_craft_1"
+	base_turf = /turf/space/dynamic
+	base_area = /area/space
+
+/obj/effect/shuttle_landmark/elyran_strike_craft/nav2
+	name = "Elyran Naval Strike Craft - Port Airlock"
+	landmark_tag = "nav_elyran_strike_craft_2"
+	base_turf = /turf/space/dynamic
+	base_area = /area/space
+
+/obj/effect/shuttle_landmark/elyran_strike_craft/transit
+	name = "In transit"
+	landmark_tag = "nav_transit_elyran_strike_craft"
+	base_turf = /turf/space/transit/south
+
+//shuttle stuff
+/obj/effect/overmap/visitable/ship/landable/elyran_shuttle
+	name = "Elyran Naval Shuttle"
+	desc = "Easily mistaken for a Wisp-class (Which it is almost identical to), this shuttle is in fact an Elyran Dromedary-class, which is an unlicensed copy of the Hephaestus Wisp-class, and one of the designs slated for retirement in the ongoing military modernization program. Hephaestus Industries has attempted to sue the Elyran government many times for the navy’s unlicensed production of its intellectual property, and Elyran courts have, unsurprisingly, always ruled in favor of the government - much to the frustration of the megacorporation, which has no other method of recourse. An inefficient design of ultra-light shuttle. Its only redeeming features are the extreme cheapness of the design and the ease of finding replacement parts. This one’s transponder identifies it as an Elyran Naval Infantry recovery shuttle."
```

Comment:
'Which' shouldn't be capitalized in the parentheses; not a complete sentence, just a fragment.

Reply:
Addressed